apps/site/src/app/demo/page.js

```diff
@@ -0,0 +1 @@
+export { default as default } from "./Demo";
```

Comment:
Nitpick: Shouldn't this be a `.ts` file?

Reply:
Suggestion [non-blocking]: It might be good to obscure this route in prod so it can only be accessible via previews.

Reply:
Regarding file type, the file was just for a reexport and not any typed code, so I didn't bother, but I suppose it's up to preference.
Regarding hiding the route, good point, but I think we should deploy once to production to verify the behavior still works when the actual domain name (instead of *.vercel.app) is applied before hiding it.